tests/cases/cancel.rs

```diff
@@ -27,13 +31,15 @@ type RouteChatHandler =
 
 #[derive(Clone)]
 struct CancelService {
+    list_feature_listener: Arc<Mutex<Option<std_mpsc::Sender<()>>>>,
```

Comment:
What about `Handler<Sender<()>>`?

Reply:
It's just a `Sender` rather than a `Box`.